ffi/tests/ffi_import.rs

```diff
@@ -0,0 +1,127 @@
+#![allow(unsafe_code, clippy::restriction, clippy::pedantic)]
+
+use std::{alloc, ops::Deref};
+
+use iroha_ffi::{ffi_import, FfiType, LocalRef, LocalSlice};
+
+iroha_ffi::def_ffi_fn! { dealloc }
```

Comment:
I need help here. I can't get the tests in ffi_import to dynamically link against dealloc which is needed because I'm transferring ownership of vectors over ffi now

Reply:
in the `repr_c.rs` module I'm calling `__dealloc` function which is only available at link time. You can see it's declared there with `decl_ffi_fn`. I would like to provide a definition for `__dealloc` function later - in this case in `ffi_import`

Reply:
Roman saved the day, I was missing imports in other test files

Reply:
Roman deserves a commendation.